pkg/kudoctl/cmd/install/install.go

```diff
@@ -176,7 +179,19 @@ func installCrds(crds *packages.PackageCRDs, kc *kudo.Client, options *Options,
 	if !instanceExists {
 		if err := installSingleInstanceToCluster(operatorName, crds.Instance, kc, options, settings); err != nil {
 			return errors.Wrap(err, "installing single instance")
-
+		}
+		if options.Wait == true {
+			for {
+				status, err := plan.PlanStatus(&plan.Options{Instance: instanceName}, settings)
+				if err != nil {
+					return errors.Wrap(err, "getting plan status")
+				}
+				if status.Pending {
+					time.Sleep(2 * time.Second)
```

Comment:
couple notes over this block:

I would consider replacing this with a for + select, and have a few channels you're selecting on. This will enable you to add timeouts in a nice, clean way. Eventually we might want to move this out to some Waiter package/func/interface as well.

Also, I'd think we'd want some sort of timeout, even if it's high (10 minutes?). 

Check out the select block here to see a bit of what I mean on that:

https://gobyexample.com/timeouts

Where this gets interesting (and maybe we pair on Monday) is you'd need some sort of goroutine-based retry for actually fetching the planStatus.

Reply:
actually, nevermind, this handles the client in the way we want it to. So we may just want to separate out the display and the logic into two separate functions. If you want to try your hand at that...that'd be a great next step before we do any refactoring into channel-based logic.